TestResultSummaryService/parsers/BenchmarkParser.js

```diff
@@ -152,6 +154,19 @@ class BenchmarkParser extends Parser {
                 benchmarkProduct: curBenchmarkProduct,
                 testData: curTestData
             } );
+            if ( curTestData["metrics"] && curTestData["metrics"].length > 0 ) {
+                for ( let l = 0; l < curTestData["metrics"].length; l++ ) {
+                    let childMetricName = curTestData["metrics"][l]["name"];
+                    let childValue = curTestData["metrics"][l]["value"];
+                    childAggData.push({name: childMetricName, value: {mean: math.mean(childValue), max: math.max(childValue), min: math.min(childValue), median: math.median(childValue), stddev: math.std(childValue), CI: BenchmarkMath.confidence_interval(childValue), iteration: 1}});
```

Comment:
Even though we'll be removing this `stddev` function in order to use the similar function from `mathjs`, we need to do a similar check in `BenchmarkParser.js` 
We need to do checks as done in our internal `stddev` function in `BenchmarkMath.js`. Before using the standard library functions, we need to make sure that all values in the array are good (i.e. not null) before we call all the math functions such as `max`, `stddev` and others. We should be passing a new validated array, which doesn't have any null values, to those functions.